notion/assets/dashboards/notion.json

```diff
@@ -0,0 +1 @@
+{"title":"Notion Dashboard","description":"A started dashboard for Notion page events","widgets":[{"id":6237632569622898,"definition":{"type":"free_text","text":"Page Events","color":"#000","font_size":"auto","text_align":"left"},"layout":{"x":0,"y":0,"width":12,"height":1}},{"id":2926389996605492,"definition":{"title":"Pages Created - past week","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1w"},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"data_source":"logs","name":"query1","indexes":["*"],"compute":{"aggregation":"count"},"group_by":[],"search":{"query":"@event.type:page.created"},"storage":"hot"}],"response_format":"timeseries","style":{"palette":"blue","line_type":"solid","line_width":"thick"},"display_type":"line"}]},"layout":{"x":0,"y":1,"width":4,"height":2}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
The dashboard should include a description on the value of the dashboard. The dashboard should also include any widgets that users will find helpful for the integration. I highly recommend taking a look at [this dashboard article](https://docs.datadoghq.com/developers/integrations/create-an-integration-dashboard/#follow-dashboard-best-practices) that goes over best practices for integration dashboards.

Reply:
As I stated earlier, this is not the end state of the dashboard and will not be used beyond our select (less than 10) customer early access program. Therefore, I do not believe we need any robust definitions or descriptions around dashboards. We will add more robust dashboarding as we better understand end user dashboarding requirements.